src/Campaigns/Blocks/CampaignCover/render.php

```diff
@@ -35,8 +35,8 @@
         src="<?php echo esc_url($campaign->image); ?>"
         alt="<?php echo esc_attr($altText); ?>"
         style="
-            width:<?php echo $width ?>px;
-            height: <?php echo $height ?>px;
+            <?php echo $widthStyle ?>
+            <?php echo $heightStyle; ?>
```

Comment:
We should add escaping to these

Reply:
Resolved: 8f61933c2a52706d4ef8e22b9e1893f1269174e6